### PR TITLE
Prizes component

### LINKED ISSUE #389: `Prizes` component
### Pre-requisites:
- [x] I looked through both [open and closed issues](../issues?utf8=✓&q=is%3Aissue) and did not find another request for the same or similar feature.

## Description

Pull the prize data from #388 and display prize data, including a header and rows for each item. If items with `isBonus == true` are included, display a separate section below the other prizes. Each item should display a description if one is provided.

```
/components/Prizes/
  index.ts (only exports `Prizes`)
  PrizeRow.tsx (with `variant = primary` prop, `secondary` for bonus prizes)
  Prizes.tsx
```

Dependent on #390

### PR DESCRIPTION
**Pull request by Ulfingrad**
## Pre-Requisites
- [x] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

## Description of Changes

Implements the prizes component. 

Of note - the bonus prize table does not align with the normal prizes table and I'm not quite sure why. Additionally, I think I misunderstood the implementation of the `prop variant` mentioned in #389, so double-checking if I did it wrong would be great as well!

screenshot1

## Related Issues

Closes #389

### CODE REVIEW

**Comment by johnkahn** [comment]
They're different sizes because they are different tables. The column widths are dependent on the contents, so if it's not the same table they'll be different. You would want to have them all be in the same table. So you would just put the two maps back to back in the `TBody`.

And then for the `variant` prop, we were meaning it to be used like this

```tsx
<PrizeRow variant={prize.isBonus ? 'primary' : 'secondary'} place={i + 1} prize={prize} />
```

Where `variant` decides whether it shows the place or not, `place` is the `index + 1` and will only be visible for `primary` variants. The table would look something like this

|  | Prize | Description |
|---|---|----|
| 1 | Prize Name | A prize description |
| 2 | Prize Name | A prize description |
| 3 | Prize Name | A prize description |
|   | Bonus Prize Name | A bonus prize description |
|   | Bonus Prize Name | A bonus prize description |

**Comment by Ulfingrad** [comment]
@johnkahn should be ready for a review when you get the chance (I'm not blocked by this getting merged so take your time if ya need it)

File: packages/web/src/components/Prizes/PrizeRow.tsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import React from 'react';
 
 interface PrizeRowProps {
   prize: Prize;
+  variant: string;
```
Comment (johnkahn) [inline packages/web/src/components/Prizes/PrizeRow.tsx:6]:
```suggestion
  variant:'primary' | 'secondary';
```

File: packages/web/src/components/Prizes/Prizes.tsx [hunk 2]
```diff
@@ -1,61 +1,34 @@
...
-        <div style={{ display: 'none' }}></div>
-      )}
-    </VStack>
+export const Prizes: React.FC<PrizeProps> = ({ prizes }) =>
+  prizes.length > 0 ? (
+    <Table variant="simple">
+      <Thead>
+        <Tr>
+          <Th>Prize Placement</Th>
+          <Th>Prizes</Th>
```
Comment (johnkahn) [inline packages/web/src/components/Prizes/Prizes.tsx:15]:
```suggestion
          <Th>Placement</Th>
          <Th>Prize</Th>
```

**Review by johnkahn — CHANGES_REQUESTED** [review]
Two quick changes, then it's good to go

**Review by johnkahn — APPROVED** [review]

**Pull request merged**